### PR TITLE
Add License file

### PR DESCRIPTION
**Pull request by alwyn-ixo**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (Zach-ixo) [inline LICENSE:1]: @alwyn-ixo This is an Apache License and not MIT, is this intended?

Comment (Zach-ixo) [reply LICENSE:1]: @alwyn-ixo just double checking here?

Comment (alwyn-ixo) [reply LICENSE:1]: Yes, as per Shaun: "Apache 2 is also open-source but requires attribution. This is not a commercial license."

Comment (alwyn-ixo) [reply LICENSE:1]: @Zach-ixo

**Review by Zach-ixo — APPROVED** [review]

**Pull request merged**